### DISCUSSION TITLE
Remove a remote tag.

### DISCUSSION THREAD

**Discussion by JimMoen** [magit/magit#4911] [Ideas]
i.e. 
For software under continuous development, it is often necessary to retag `docker-latest`
So we need remove a remote tag, and re-push the tag (on newest commit) to remote.

Or magit already has this feature, I just can't find it?
I can't find a magit function for this command.

```bash
git push --delete origin [tag-name]
```

**Comment by tarsius** [comment]
You can either delete the tag using an explicit refspec `p r` `origin RET` `:refs/tags/the-tag RET` (`git push -v origin \:refs/tags/test`), or force overriding the existing tag on push using `p - f T` `the-tag RET` (`git push --force-with-lease origin the-tag`), or delete the local tag and then prune using `t p`.
